packages/admin/src/pages/build-application/[applicationId]/[sectionId]/question-options.page.tsx

```diff
@@ -60,15 +60,25 @@ export const getServerSideProps: GetServerSideProps = async ({
     req,
     res,
     async (body: any) => {
-      options = body.options;
+      options = Object.keys(body).reduce((array, key) => {
```

Comment:
I think there is a lot going in here now and might be more readable to split into functions. 

For example one function for processing the options and another for processing the body.